eventsources/sources/github/start.go

```diff
@@ -197,6 +198,11 @@ func (el *EventListener) StartListening(ctx context.Context, dispatch func([]byt
 			Token: apiTokenCreds.secret,
 		}
 
+		httpClient := PATTransport.Client()
```

Comment:
Minor - could you move it down after `logger.Info("setting up client for GitHub...")`?